src/applications/personalization/profile/tests/e2e/address-validation/low-confidence.cypress.spec.js

```diff
@@ -0,0 +1,48 @@
+import { setUp } from 'applications/personalization/profile/tests/e2e/address-validation/setup';
+
+describe('Personal and contact information', () => {
+  describe('when entering a low confidence address that needs confirmation', () => {
```

Comment:
could this be changed to something like `when getting a single suggestion with a confidence score <90`...